QMLModule for kirigami is a runtime dependency, don't check it at buildtime.
Fixes cmake's configure step failure in cross to armv{6,7}l-*
diff --git a/CMakeLists.txt b/CMakeLists.txt
index bd89fa1..54dbfc3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -65,7 +65,6 @@ find_package(KF6 REQUIRED COMPONENTS
     QQC2DesktopStyle
 )
 
-ecm_find_qmlmodule(org.kde.kirigami REQUIRED)
 
 qt_policy(SET QTP0001 NEW)
 
